src/charm.py

```diff
@@ -325,7 +325,7 @@ def _render_conf_file(self) -> str:
             cookie_secrets=[self._get_secret()],
             log_level=self._log_level,
             mappers_path=str(self._mappers_dir_path),
-            default_browser_return_url=self._get_login_ui_endpoint_info("default_url"),
+            default_browser_return_url=self._get_login_ui_endpoint_info("login_url"),
```

Comment:
We should either use the `default_url` here or remove it from the databag (see https://github.com/canonical/identity-platform-login-ui-operator/pull/60/files#r1314575045)

Reply:
done